server/CompositeStorage/lib/index.ts

```diff
@@ -0,0 +1,31 @@
+import {
+  Storage,
+  IndexStorage,
+  ComponentsStorage,
+  Component,
+  File,
+  Index,
+  ComponentTree,
+  ComponentGetter,
+  Maybe
+} from '@dynamico/driver';
```

Comment:
Why are we getting these types out of the driver and not the common types?

Reply:
Yeah it's kind of weird... Not sure what's the correct structure here is but I wanted some package to export the types and not publish a `@types/dynamico` package. I'll play with it some more to find the best setup...